components/connector/docs/08-02-maintain-secure-connection-with-compass.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Maintain a secure connection with Compass
+type: Tutorials
+---
+
+After you have established a secure connection with Compass, you can fetch the configuration details and renew the client certificate before it expires.  
+This guide shows you how to do that.
+
+## Prerequisites
+
+- [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
+- Established secure connection with Compass
+
+> **NOTE**: To see how to establish a secure connection with Compass, see [this](08-01-establish-secure-connection-with-compass.md) document. 
+
+<!--- TODO: link in the note above --->
+
+## Get the CSR information and configuration details from Kyma using the client certificate 
+
+To fetch the configuration, make a call to the Certificate-Secured Connector URL using the client certificate. 
+The Certificate-Secured Connector URL is the `certificateSecuredConnectorURL` obtained when establishing a secure connection with Compass. 
+Send this query with the call:
+```
+query {
```

Comment:
I got an idea! It's a suggestion so just consider doing that. I found that markdown (at least on the GitHub site) supports highlighting GraphQL code! It could look as cool as the example below:

```graphql
query {
    result: configuration {
        certificateSigningRequestInfo { 
            subject 
            keyAlgorithm 
        }
        managementPlaneInfo { 
            directorURL 
        }
    }
}
```

It's as easy as adding `graphql` after first three backticks. It would be:
```
```graphql
...
```